pkg/provision/mbus/api.go

```diff
@@ -85,6 +88,41 @@ func (n *Deployments) listPublicIps(ctx context.Context, _ []byte) (interface{},
 	return ips, nil
 }
 
+type listPrivateIpsArgs struct {
+	NetworkName string `json:"network_name"`
```

Comment:
```suggestion
	NetworkName gridtypes.Name `json:"network_name"`
```

If you do this you won't need the cast in the code anymore